projects/pgai/pgai/semantic_catalog/vectorizer/vectorizer.py

```diff
@@ -79,6 +79,7 @@ def create(
 
 
 def embedding_config_from_dict(config: dict[str, Any]) -> EmbeddingConfig:
+    config["config_type"] = "embedding"
```

Comment:
instead of modifying the dict, could we do `OllamaConfig(config_type="embedding", **config)`? Does that work?

Reply:
Done. Had the same exact thought right after opening this PR, had just gotten to force push up my branch.